eth/backend.go

```diff
@@ -385,6 +386,13 @@ func (s *Ethereum) APIs() []rpc.API {
 			Namespace: "net",
 			Service:   s.netRPCService,
 		},
+		// CELO specific API backend.
+		// For methods in the backend that are already defined (match by name)
+		// on the eth namespace, this will overwrite the original procedures.
+		{
+			Namespace: "eth",
```

Comment:
Should we remove the `eth` namespace above to make it clearer that this takes over? It does override the entry above, right?

Reply:
No, it only overwrites (re-registers the callback-lookup for) the methods that are implemented on the CeloAPI struct. The idea was to have a place where Celo specific overwrites can happen if so desired. They could either be wrappers or implement completely new behavior.

Reply:
Ok, makes sense.